### PR TITLE
Updated coala banner

### LINKED ISSUE #436: Update coala banner on README
The banner on [coala/coala](https://github.com/coala/coala) has changed into a more modern look to match the new website. I think the same thing should be done as well on this repository for uniformity.

### PR DESCRIPTION
**Pull request by yashasingh**
Updated coala banner on README.md 
This PR fixes #436

### CODE REVIEW

**Comment by RaiVaibhav** [comment]
Please add `Fixes` in commit message also , and [please read](coala.io/newcomer) you will to know about many things

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,5 @@
-.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+.. image:: https://cloud.githubusercontent.com/assets/5716520/24838296/a9cf5f04-1d45-11e7-855c-47b816ce1e09.png
+    :target: https://coala.io/
```
Comment (RaiVaibhav) [inline README.rst]: Please make it to  `https://coala.io` instead `https://coala.io/`

Comment (RaiVaibhav) [reply README.rst]: It also works fine  without any change but see the diffrence between these two http://api.coala.io/en/latest/Developers/Newcomers_Guide.html/  and http://api.coala.io/en/latest/Developers/Newcomers_Guide.html

Hope you will get my point

Comment (jayvdb) [reply README.rst]: That is irrelevant

**Comment by jayvdb** [comment]
The content of your patch is fine; please amend your commit message
http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
This type of change is "Closes", not a "Fixes", as the current version is not broken.

**Comment by yashasingh** [comment]
@jayvdb I have updated my commit message as per instructions. Please review.
Thank you.

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/485/commits/cb7d408a6e91450eeaf6ebeaac590e1e4f5b4205

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**